src/software/ai/evaluation/calc_best_shot.cpp

```diff
@@ -163,10 +163,9 @@ std::optional<Shot> calcMostOpenDirectionFromCircleObstacles(
     // If there are no obstacles, return the center of the Segment and the shot angle
     if (obstacles.size() == 0)
     {
-        const Point center_of_segment =
-            getPointsMean({segment.getSegStart(), segment.getEnd()});
+        const Point center_of_segment = segment.midPoint();
         const Angle angle_of_entire_segment =
-            ((segment.getSegStart() - origin)
+            ((segment.getStart() - origin)
```

Comment:
Since there is toVector, there is a direction

Reply:
I don't necessarily agree that a Segment should have direction because so for toVector() convention. IMO, a Segment is directionless and maybe we should look into our uses of toVector() and see if it matters.

Reply:
OK that makes sense, but that would mean that we don't have any sense of start or end, so should getStart and getEnd be deleted as well?

Reply:
Yea I agree this is worth investigating, but as a separate ticket since changing how end points work is out of scope for this PR

Reply:
Ticket created at https://github.com/UBC-Thunderbots/Software/issues/1603